python/lsst/validate/base/metric.py

```diff
@@ -55,6 +142,12 @@ class Metric(JsonSerializationMixin):
     description = None
     """Short description of the metric (`str`)."""
 
+    unit = None
```

Comment:
Ah, great! I was wondering about how exactly to manage this, so I'm glad you already came up with something. I'll see if I can switch it over to that.